angles:
  left: 30
  middle: 0
  right: -30
raw:
  raw_left: 2000
  raw_middle: 2600
  raw_right: 3200
